### ISSUE TITLE
Lazy loading on `t`/`T`/`f`/`F` stopped working

### ISSUE THREAD

**Issue by mawkler** [jinh0/eyeliner.nvim#53]
Following up the discussion in https://github.com/jinh0/eyeliner.nvim/issues/46. Here's a minimal config with the lazy loading issue:

<details>
<summary><b>Click to expand</b></summary>

```lua
local lazypath = vim.fn.stdpath('data') .. '/lazy/lazy.nvim'
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    'git',
    'clone',
    '--filter=blob:none',
    'https://github.com/folke/lazy.nvim.git',
    '--branch=stable', -- latest stable release
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)

vim.keymap.set('n', '<c-q>', vim.cmd.quit)
vim.api.nvim_set_hl(0, 'LeapBackdrop', { link = 'Comment' })

local mode = { 'n', 'x', 'o' }
require('lazy').setup({
  'jinh0/eyeliner.nvim',
  keys = {
    { 't', mode = mode },
    { 'T', mode = mode },
    { 'f', mode = mode },
    { 'F', mode = mode },
  },
  opts = {
    highlight_on_key = true,
  }
})
```

</details>

The key mappings don't get created after lazy loading on `t`/`T`/`f`/`F`. I seeem to have to switch to another buffer and back for the key mappings to appear.

**Comment by jinh0** [comment]
Hey @mawkler , I believe I've fixed it! Turns out I forgot to enable eyeliner keybinds on setup, it was only enabling keybinds on the BufEnter autocommand.  It's fixed in the latest commit 587c096de0cc3e97ae0993b5b7b9da9b64f72cd4 .

**Comment by mawkler** [comment]
Nice catch! It seems to work for me. Thank you! 🙂
